spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -476,6 +476,37 @@ def expect_changes_made_by(admin, log_items)
     end
   end
 
+  describe "POST #destroy_user_creations" do
```

Comment:
You probably don't have to create a new `describe` block as the examples above are testing the same method. You can add the `it` block within the context "when user is banned` below [this line](https://github.com/otwcode/otwarchive/pull/4719/files#diff-0db7b6f87501fc226438013cfe7b2f9bc5fa7fd2ee0a387a91ddb591534c1af2R466).

Reply:
@ceithir wanted a completely separate test.

Reply:
A distinct test i.e. `it`, with its own relevant context indeed. They can definitely share the same `describe` though.

Reply:
I misunderstood then. If I add it back in to the original test as its own it, would I need line 499? Since this is initiated in the last test of the first destroy_user_creations?